site/docs/src/json/scim/group-create-request.json

```diff
@@ -0,0 +1,12 @@
+{
+  "externalId" : "2819c223-7f76-453a-919d-413861904646",
+  "displayName": "Sales Reps",
+  "members": [
+    {
+      "displayName": "John Doe",
+      "$ref": "https://fusionauth.io/api/scim/v2/Users/902c246b-6245-4190-8e05-00816be7344a",
```

Comment:
I wouldn't use our actual URL here of `https://fusionauth.io/` this is a production URL. Can we use something else? We use `login.piedpiper.com` quite a bit in docs, so that is an option.